README.md

```diff
@@ -4,165 +4,171 @@ The ATAT Web API is the internal application programming interface that supports
+
+### Deploy Shared Infrastructure
 ---
+This monorepo currently contains a single Application (ATAT Portfolio Drafts API), but may eventually contain multiple Applications which all share some common infrastructure pieces. The following instructions describe their deployment:
+[Terraform](https://www.terraform.io/) will be used to deploy shared infrastructure in Microsoft Azure.
+Examples of such shared resources:
+* Key Vault
+* Storage Account
+* Cosmos DB Account and Database 
+* Azure Service Bus
 
-#### Login to Azure and set subscription
-```
-az login
-az account set -s <subscription-id>
-```
-
-
-## Shared Infrastructure
-Shared infrastructure for the PoC can be found at `./poc/shared_infrastructure`
-
-### Install dependencies
-For a detailed tutorial, follow the guidance here: [Install Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/azure-get-started)
-
-### Local Deployment
-
-Navigate to the `./poc/shared_infrastructure` directory.
+The following is a list of `main.tf` files and their locations which can be used with Terraform to deploy Shared Infrastructure:
+* `./provisioning/shared_infrastructure/main.tf`
 
-Ensure you are logged into Azure with `az login`.
+#### Prerequisites
+1. Navigate to a directory containing a `main.tf` file.
+2. Prepare a file `terraform.tfvars` from file `terraform.tfvars.template` in the same directory (if available)
+3. Ensure you are logged in to Azure with `az login`
+4. Set your desired subscription with `az account set -s <subscription-id>`.  You can view the selected subscription with `az account show` 
 
-#### Terraform Init
+#### Terraform Steps
 
-You will need the name of the storage account where state will be stored as well as the
-resource group that the storage account is in. Those will be called `$AZURE_STORAGE_ACCOUNT` and
-`$AZURE_STORAGE_RESOURCE_GROUP` respectively in any subsequent steps.
+##### Initialization
+You will need the name of a storage account where Terraform [state](https://www.terraform.io/docs/language/state/index.html) will be stored as well as the [resource group](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resource-groups-portal#what-is-a-resource-group) that contains it. Those will be called `$AZURE_STORAGE_ACCOUNT` and `$AZURE_STORAGE_RESOURCE_GROUP` respectively in any subsequent steps.
 
 ```
 terraform init -backend-config=resource_group_name=$AZURE_STORAGE_RESOURCE_GROUP -backend-config=storage_account_name=$AZURE_STORAGE_ACCOUNT
 ```
 
-#### Retrieve .tfstate from Azure
+##### Select a workspace
 
-Select the correct workspace inside the container on Azure
+Select the desired workspace.
+Terraform state is stored by default in a file named _terraform.tfstate_.  In this case it is stored remotely inside the storage container on Azure.
 
 ```
-terraform workspace select <name of workspace>
+terraform workspace select <your-workspace-name>
```

Comment:
Should we add a `terraform new <your-workspace-name>` step before this step if they haven't made one yet?

Reply:
I like that idea.

Reply:
Added.